extensions/aproc/proc/access/manager.py

```diff
@@ -68,18 +68,23 @@ def get_storage_parameters(href: str):
 
         return storage.get_storage_parameters()
 
+    @staticmethod
+    def check_local_path_authorized(href: str):
```

Comment:
It seems that by "authorized", you mean "writable"? if so, it could be "is_writable"?